.pre-commit-config.yaml

```diff
@@ -5,14 +5,12 @@ repos:
       - id: trailing-whitespace
       - id: end-of-file-fixer
       - id: check-yaml
-
   - repo: https://github.com/PyCQA/isort
     rev: 5.8.0
     hooks:
       - id: isort
-
-  - repo: https://github.com/python/black
-    rev: 21.5b1
+  - repo: https://github.com/psf/black
```

Comment:
Wondering why we had `python/black` here? I believe that has just been renamed, as the link points to `...psf/black` now.